vector/src/main/java/im/vector/riotx/features/home/room/detail/timeline/item/MessageInformationData.kt

```diff
@@ -31,7 +31,8 @@ data class MessageInformationData(
         val showInformation: Boolean = true,
         /*List of reactions (emoji,count,isSelected)*/
         var orderedReactionList: List<ReactionInfoData>? = null,
-        var hasBeenEdited: Boolean = false
+        var hasBeenEdited: Boolean = false,
```

Comment:
we should only get val on this class, could you please change that on the 3 fields?

Reply:
done